openfact.gemspec

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = 'openfact'
-  spec.version       = '5.0.0'
+  spec.version       = '5.1.0'
```

Comment:
cant you do `Facter.VERSION` here?

Reply:
@smortex tried that and that caused different issues that we still need to investigate :(

Reply:
or Was ist Facter::VERSION. saw this somewhere. so you only have on version string to update

Reply:
https://github.com/voxpupuli/r10k/blob/main/r10k.gemspec